.travis.yml

```diff
@@ -10,12 +10,14 @@ matrix:
   - php: 7.4
     env: ANALYSIS='true'
   - php: nightly
+    env: IGNOREPHP='true'
   allow_failures:
   - php: nightly
 
 before_script:
   - if [[ "$ANALYSIS" == 'true' ]]; then composer require php-coveralls/php-coveralls:^2.1.0 ; fi
-  - composer install -n
+  - if [[ "IGNOREPHP" == 'true' ]]; then composer install -n --ignore-platform-reqs; fi
+  - if [[ "IGNOREPHP" != 'true' ]]; then composer install -n; fi
```

Comment:
Why do we need to ignore platform requirements? Is there another way to fix this?